source/assets/css/components/_callouts.scss

```diff
@@ -15,19 +15,12 @@
   }
 
   &--function {
-    padding: {
-      top: 0;
-      bottom: .25rem;
-    };
+  padding: 0 2.25rem .25rem;
 
     .signature {
       // Make sure permalinks are visible.
       overflow: visible;
-
-      margin: {
-        right: -1rem;
-        left: -1rem;
-      };
+      margin-left: -2.25rem;
```

Comment:
can we make these changes in the larger breakpoint only? right now in production its even both sides (margins & paddings) but this change (meant to adjust for larger size) will look wrong on smaller screen sizes (like a mobile web size).

Reply:
Thanks, fixed.